package.json

```diff
@@ -1,54 +1,75 @@
 {
   "name": "cf-nodejs-logging-support",
-  "version": "6.13.2",
+  "version": "7.0.0-beta.0",
   "description": "Logging tool for Cloud Foundry",
   "keywords": [
     "logging",
     "cf",
     "perfx"
   ],
+  "main": "build/main/index.js",
+  "types": "build/main/index.d.ts",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/SAP/cf-nodejs-logging-support"
   },
   "license": "Apache-2.0",
   "author": {
-    "name": "Christian Dinse, Nicklas Dohrn"
+    "name": "Federico Romagnoli, Christian Dinse"
   },
   "homepage": "https://sap.github.io/cf-nodejs-logging-support/",
-  "main": "./index.js",
-  "engines": {
-    "node": ">=8.0.0"
+  "scripts": {
+    "build": "npm run build-json-schema && tsc -p tsconfig.json",
+    "start": "npm run build-json-schema && tsc -p tsconfig.json && node build/main/index.js",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "npm run start && mocha 'src/test/**/*.test.js'",
+    "test:lint": "eslint src --ext .ts",
+    "test-performance": "mocha 'src/performance-test/*.test.js'",
+    "build-json-schema": "typescript-json-schema 'src/lib/config/interfaces.ts' ConfigObject --noExtraProps --required --out 'src/lib/config/config-schema.json'"
   },
-  "dependencies": {
-    "json-stringify-safe": "^5.0.1",
-    "jsonwebtoken": "^8.5.1",
-    "uuid": "^8.3.2"
+  "engines": {
+    "node": ">=10"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "import-fresh": "^3.2.1",
-    "mocha": "^7.1.1",
-    "mock-require": "^3.0.3",
-    "node-mocks-http": "^1.8.1",
-    "nyc": "^15.0.1",
-    "rewire": "^4.0.1",
-    "sinon": "^9.0.1",
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
-  },
-  "scripts": {
-    "test": "mocha -- --timeout 10000",
-    "coverage": "nyc --reporter=lcov --reporter=text-summary npm run test"
+    "@types/express": "^4.17.13",
+    "@types/node": "^17.0.45",
+    "@types/restify": "^8.5.4",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^3.10.1",
+    "chai": "^4.3.6",
+    "eslint": "^8.20.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-functional": "^3.0.2",
+    "eslint-plugin-import": "^2.22.0",
+    "import-fresh": "^3.3.0",
+    "mocha": "^9.2.2",
+    "node-mocks-http": "^1.11.0",
+    "rewire": "^6.0.0",
+    "supertest": "^6.2.2",
+    "ts-node": "^9.0.0",
+    "typedoc": "^0.22.13",
+    "typescript": "^4.0.2",
+    "typescript-json-schema": "^0.53.0"
   },
   "files": [
-    "/.reuse/",
-    "/core/",
-    "/logger/",
-    "/trace/",
-    "/winston/",
-    "config.js",
-    "index.js",
-    "LICENSE"
-  ]
+    "build/main",
+    "build/module",
+    "!**/*.spec.*",
+    "!**/*.json",
```

Comment:
This rule excludes our config json files, doesn't it?

Reply:
I will deleted these negation rules since our config files are already included in the build folder. If I am not wrong we only need to upload the `build` folder, `LICENSE`, `README.md` and `package.json` (which is always included regardless of this setting).